### PR TITLE
Cypress test. Review pipeline feature.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Cypress test. Review pipeline feature. 
Adding the necessary css classes for the test. Adding new cypress commands. Adaptation of some fixed tests.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by ActiveChooN — DISMISSED** [review]

**Comment by nmanovic** [comment]
@dvkruchinin , please resolve conflicts.

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx [hunk 1]
```diff
@@ -55,22 +55,25 @@ export default function LabelsListComponent(): JSX.Element {
                 <Row justify='start' align='middle'>
                     <Col>
                         <Tooltip title='Find the previous frame with issues'>
-                            <LeftOutlined {...dinamicLeftProps} />
+                            <LeftOutlined className='cvat-objects-sidebar-issues-list-header-left' {...dinamicLeftProps} />
```
Comment (bsekachev) [inline cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx:58]: cvat-issues-sidebar-previous-frame

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx [hunk 2]
```diff
@@ -55,22 +55,25 @@ export default function LabelsListComponent(): JSX.Element {
...
                         </Tooltip>
                     </Col>
                     <Col offset={1}>
                         <Tooltip title='Find the next frame with issues'>
-                            <RightOutlined {...dinamicRightProps} />
+                            <RightOutlined className='cvat-objects-sidebar-issues-list-header-right' {...dinamicRightProps} />
```
Comment (bsekachev) [inline cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx:63]: cvat-issues-sidebar-next-frame

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx [hunk 3]
```diff
@@ -55,22 +55,25 @@ export default function LabelsListComponent(): JSX.Element {
...
                         </Tooltip>
                     </Col>
                     <Col offset={3}>
                         <Tooltip title='Show/hide all the issues'>
                             {issuesHidden ? (
                                 <EyeInvisibleFilled
+                                    className='cvat-objects-sidebar-issues-list-header-hidden'
```
Comment (bsekachev) [inline cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx:70]: cvat-issues-sidebar-hidden-issues

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx [hunk 4]
```diff
@@ -55,22 +55,25 @@ export default function LabelsListComponent(): JSX.Element {
...
                                     onClick={() => dispatch(reviewActions.switchIssuesHiddenFlag(false))}
                                 />
                             ) : (
-                                <EyeOutlined onClick={() => dispatch(reviewActions.switchIssuesHiddenFlag(true))} />
+                                <EyeOutlined
+                                    className='cvat-objects-sidebar-issues-list-header-shown'
```
Comment (bsekachev) [inline cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx:75]: cvat-issues-sidebar-shown-issues

File: tests/cypress/support/commands_review_pipeline.js [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+    cy.get('.cvat-task-details-user-block').within(() => {
+        cy.get('.cvat-user-search-field').click();
+    });
+    cy.get('.ant-select-dropdown')
+        .not('.ant-select-dropdown-hidden')
+        .contains(new RegExp(`^${user}$`, 'g'))
+        .click();
+});
+
+Cypress.Commands.add('assignJobToUser', (jobNumber, user) => {
```
Comment (bsekachev) [inline tests/cypress/support/commands_review_pipeline.js:17]: I suppose it is jobID, not a jobNumber

File: tests/cypress/support/commands.js [hunk 6]
```diff
@@ -102,24 +102,28 @@ Cypress.Commands.add('openTask', (taskName) => {
...
+    cy.server().route(method, '/api/v1/jobs/**').as('saveJob');
     cy.get('button').contains('Save').click({ force: true });
-    cy.wait('@sendLogs').its('status').should('equal', 201);
+    cy.wait('@saveJob').its('status').should('equal', status);
 });
 
-Cypress.Commands.add('openJob', (jobNumber = 0) => {
-    let tdText = '';
-    cy.get('.ant-table-tbody')
+Cypress.Commands.add('getJobNum', (jobNumber) => {
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:111]: jobID

File: tests/cypress/support/commands.js [hunk 7]
```diff
@@ -102,24 +102,28 @@ Cypress.Commands.add('openTask', (taskName) => {
...
         .eq(0)
         .invoke('text')
         .then(($tdText) => {
-            tdText = Number($tdText.match(/\d+/g)) + jobNumber;
-            cy.get('.ant-table-tbody').contains('a', `Job #${tdText}`).click();
+            return Number($tdText.match(/\d+/g)) + jobNumber;
         });
+});
+
+Cypress.Commands.add('openJob', (jobNumber = 0) => {
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:123]: jobID

File: tests/cypress/support/commands_review_pipeline.js [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+            .find('.cvat-job-assignee-selector')
+            .click();
+    });
+    cy.get('.ant-select-dropdown')
+        .not('.ant-select-dropdown-hidden')
+        .contains(new RegExp(`^${user}$`, 'g'))
+        .click();
+});
+
+Cypress.Commands.add('checkJobStatus', (jobNumber, status, assignee, reviewer) => {
```
Comment (bsekachev) [inline tests/cypress/support/commands_review_pipeline.js:31]: jobID

Comment (dvkruchinin) [reply tests/cypress/support/commands_review_pipeline.js]: Done

File: tests/cypress/support/commands.js (hunk 7 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js]: Done

File: tests/cypress/support/commands.js (hunk 6 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js]: Done

File: tests/cypress/support/commands_review_pipeline.js (hunk 5 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands_review_pipeline.js]: Done

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx (hunk 4 above)
Comment (dvkruchinin) [reply cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx]: Done

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx (hunk 3 above)
Comment (dvkruchinin) [reply cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx]: Done

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx (hunk 2 above)
Comment (dvkruchinin) [reply cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx]: Done

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx (hunk 1 above)
Comment (dvkruchinin) [reply cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx]: Done

**Review by bsekachev — APPROVED** [review]

**Pull request merged**